app/assets/styles/map.scss

```diff
@@ -1,5 +1,10 @@
 $accent-color: #2C9FA3;
 $font-choice: Roboto;
+$font-size: 14px;
+
+.ui.menu {
```

Comment:
in general when styling semantic ui components i would add an extra class for that specific element because if in the future we add more `ui menu` items elsewhere in the app we might want to style it differently